README.md

```diff
@@ -1 +1,63 @@
-# Hello-Microverse
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a simple project that display a hello microverse message.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+### Key Features
+- **[When clicked it prints "Hello Microverse" in aquamarine color]**
+## :rocket: Live Demo
+- [No Live Demo Link]
+## :computer: Getting Started
+prints Hello Microverse in default browser.
+### Prerequisites
+Clone this repository to your desired folder and open html file.
```

Comment:
The prerequisites section should contain the things a new user should know before being able to use the project. I recommend you include this section correctly to make your readme more effective in instructing a new user using the above-linked readme as a reference.